### PR TITLE
[492] SSL certificates guidance

### PR DESCRIPTION
**Pull request by saliceti**
## What
Migrated from https://dfedigital.atlassian.net/wiki/spaces/BaT/pages/1540784287/SSL+certificates

Details reviewed today

### CODE REVIEW

File: source/infrastructure/security/ssl-certificates/index.html.md.erb [hunk 1]
```diff
@@ -7,4 +7,137 @@ review_in: 6 months
...
+  - Validity: 12 months
+  - Content type: PKCS #12
+  - Lifetime action type: Automatically renew at a given number of days before expiry
+  - Number of days: 30
+- Click Create
+- Click Certificate Operation and wait for result
+
+## Renewals
+
+The certificates are monitored and owners are notified via email before expiry by either GlobalSign or the _Platform support_ team.
```
Comment (sfawcett123) [inline source/infrastructure/security/ssl-certificates/index.html.md.erb:125]: Could Platform Support Team not be a Link ?

File: source/infrastructure/security/ssl-certificates/index.html.md.erb [hunk 2]
```diff
@@ -7,4 +7,137 @@ review_in: 6 months
...
+
+## Check certificate
+
+In order to view certificate expiry details such as expiry date etc, you can run the following command to download the certificate currently being used on a website. In this example, we are focusing on _https://find-postgraduate-teacher-training.education.gov_.uk :
```
Comment (sfawcett123) [inline source/infrastructure/security/ssl-certificates/index.html.md.erb:129]: Now I am being so picky but I havent had coffee :)

etc should be etc. and therefore you should have etc.,

**Review by sfawcett123 — COMMENTED** [review]
since I have not had to do this process I cannot comment on that aspect

File: source/infrastructure/security/ssl-certificates/index.html.md.erb (hunk 1 above)
Comment (saliceti) [reply source/infrastructure/security/ssl-certificates/index.html.md.erb]: Linked to support section

File: source/infrastructure/security/ssl-certificates/index.html.md.erb (hunk 2 above)
Comment (saliceti) [reply source/infrastructure/security/ssl-certificates/index.html.md.erb]: fixed thanks

**Review by sfawcett123 — APPROVED** [review]

**Comment by saliceti** [comment]
[tech guidance: SSL certificates](https://trello.com/c/4ayv6Zfp/492-tech-guidance-ssl-certificates)

**Pull request merged**